uint8 pin
uint32 period_ms
float32 percent
---
uint8 result
